index.html

```diff
@@ -0,0 +1,1117 @@
+                </div>
+                <div class="col-lg-9">
+                  <p>"You wasted precious months when you could've taken action to protect Americans & Illinoisans,"
+                    Pritzker
+                    said</p>
+                </div>
+              </div>
+              <div class="col-lg-12 d-flex list-item mt-md-4 flex-column mb-5">
+                <div class="d-flex">
+                  <img
+                    src="https://d.newsweek.com/en/full/1575559/hand-sanitizer.webp?w=95&h=95&f=af5e9935bbff7640af192a9ea3335c17"
+                    alt="" class="w-100 h-100 col-md-4 col-lg-3">
+                  <h5>Global Coronavirus Death Toll Now Exceeds 11,500 in the year 2020 try jdhu</h5>
+                </div>
+                <div class="col-lg-9">
+                  <p>Hospitals struggle to cope in Italy, bars and restaurants in the U.K. have been closed, and the
+                    European
+                    Union allows member states to spend as much as they need to combat the virus.</p>
+                </div>
+              </div>
+              <div class="col-lg-12 d-flex list-item mt-md-5 mb-md-4 flex-column mb-5 mt-5">
+                <div class="d-flex">
+                  <img
+                    src="https://d.newsweek.com/en/full/1575683/cameron-van-der-burgh.webp?w=95&h=95&f=f32400a2feb313f893bde85415bf19ee"
+                    alt="" class="w-100 h-100 col-md-4 col-lg-3">
+                  <h5>Olympic Champion Swimmer Diagnosed With Coronavirus Calls It 'Worse Virus'</h5>
+                </div>
+                <div class="col-lg-9">
+                  <p>Cameron van der Burgh, the Olympic gold medalist in the 100-meter breaststroke in 2012 and silver
+                    medalist in 2016, talks about his physical struggles during his 14th day in battling coronavirus.
+                  </p>
+                </div>
+              </div>
+              <div class="col-lg-12 d-flex list-item mt-md-5 flex-column mb-5 mt-5 pt-5 pb-5">
+                <div class="d-flex">
+                  <img
+                    src="https://d.newsweek.com/en/full/1575653/mark-desaulnier.webp?w=95&h=95&f=a5f032524f180168dbb027d89c86a6cd"
+                    alt="" class="w-100 h-100 col-md-4 col-lg-3">
+                  <h5>Congressman Mark DeSaulnier in Critical Condition With Pneumonia</h5>
+                </div>
+                <div class="col-lg-9">
+                  <p>The doctors are doing everything they can to care for the Congressman," the representative's
+                    office
+                    said
+                    in a statement.</p>
+                </div>
+              </div>
+            </section>
+            <section id="culture-travel-mobile" class="row d-md-none mt-5 pt-4">
+              <h2 class="col-12">CULTURE & TRAVEL</h2>
+              <div class="col-md-4 col-lg-4">
+                <img
```

Comment:
Check this section. Your font-size is not the same and the font colors also need some work.
Check the badges on the images, too and add them.
On mobile screens, your  section has no title.

Yours:
![Screenshot from 2020-04-13 08-06-42](https://user-images.githubusercontent.com/51926937/79122645-d956db00-7d5d-11ea-8530-ae9e6aeaf9ea.png)

Original:
![Screenshot from 2020-04-13 08-06-49](https://user-images.githubusercontent.com/51926937/79122653-dd82f880-7d5d-11ea-985a-8023d7d29acc.png)